.release-please-manifest.json

```
@@ -80,5 +80,6 @@
   "plugins/source/oracle": "1.0.1",
   "plugins/source/oracle+FILLER": "0.0.0",
   "plugins/source/alicloud": "1.0.0",
-  "plugins/source/alicloud+FILLER": "0.0.0"
+  "plugins/source/alicloud+FILLER": "0.0.0",
+  "plugins/destination/mssql": "1.0.0"
```

```suggestion
  "plugins/source/alicloud+FILLER": "0.0.0"
```

https://github.com/cloudquery/cloudquery/blob/main/release-please-config.json is the file to update, not this one (this file is used by our bot to determine the next version so putting `1.0.0` here will generate a release PR for `1.0.1`)

---

upd in #6973